examples/uplink_set.go

```diff
@@ -9,31 +9,20 @@ import (
 
 func main() {
 	var (
-		ClientOV        *ov.OVClient
-		existing_uplink = "upset66"
-		new_uplink      = "upset77"
-		upd_uplink      = "upset88"
-		del_uplink      = "upset88"
+		ClientOV   *ov.OVClient
+		new_uplink = "test_new"
+		upd_uplink = "test_update"
 	)
 	ovc := ClientOV.NewOVClient(
 		os.Getenv("ONEVIEW_OV_USER"),
 		os.Getenv("ONEVIEW_OV_PASSWORD"),
 		os.Getenv("ONEVIEW_OV_DOMAIN"),
 		os.Getenv("ONEVIEW_OV_ENDPOINT"),
 		false,
-		800,
+		1000,
 		"")
-	ovVer, _ := ovc.GetAPIVersion()
-	fmt.Println(ovVer)
-
-	fmt.Println("#................... Uplink-Set by Name ...............#")
-	uplink_set, err := ovc.GetUplinkSetByName(existing_uplink)
```

Comment:
Why is this GetUplinkSetByName example is removed?

Reply:
Its present below in line 66. So removed as it is redundant.